tests/conftest.py

```diff
@@ -328,6 +332,50 @@ def post_editor_data(client, data, editors):
                             license = 'CC BY-SA 4.0'
                         ))
 
+class GCSTestHelper:
+    GCS_PROJECT = 'sunlit-realm-131518'
+    GCS_NUMBER = '997553930042'
+    GCS_TEST_BUCKET = 'openreview-expertise'
+    GCS_TEST_ROOT = 'jobs-test'
+    
+    @staticmethod
+    def init_test_bucket():
+        client = storage.Client()
+        bucket = client.bucket(GCSTestHelper.GCS_TEST_BUCKET)
+        
+        # Test bucket existence and permissions
+        if bucket.exists():
+            GCSTestHelper.cleanup_job_files(bucket)
+            return bucket
+        return None
+    
+    @staticmethod
+    def cleanup_job_files(bucket):
+        """Clean up job files from the bucket"""
+        if bucket is None:
+            return
+        
+        # Handle both real GCS buckets and mock buckets
+        blobs = list(bucket.list_blobs(prefix=f"{GCSTestHelper.GCS_JOBS_FOLDER}/"))
+        for blob in blobs:
+            blob.delete()
```

Comment:
Do they need to be deleted?

Reply:
Should we keep old test data in the bucket and just control the files with a lifecycle?

Reply:
Yeah, we can do that.